### DISCUSSION TITLE
I'm stuck with 'many's

### DISCUSSION THREAD

**Discussion by Kardiem** [ANLAB-KAIST/KENSv3#49] [Project Q&A 1. Application Layer (Socket)] [answered]
I got pass with one communication on test,
i'm stuck with dealing multiple connections... i can't think any direction to solve. Can i get any direction of implementation or hints?
(Is it using fork for multiple process...?? i'm not confident with that way...)

**Comment by Pusnow** [comment, marked as answer]
You must never use `fork()`.

You should handle clinets one-by-one manner.

accept -> read -> write -> accept -> ...

**Reply by antaechan** [reply]
thanks This comments is very helpful to me
